styles.css

```diff
@@ -1,556 +1,526 @@
+  .touch,
+  .connect {
+    width: 40%;
+  }
 
-.email {
-  font-family: 'Inter', sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 20px;
-  line-height: 32px;
-  text-align: center;
-  margin-top: -55px;
-  color: #42526e;
-}
+  .bar-container {
+    display: none;
+  }
 
-#four {
-  content: "";
-  position: relative;
-  width: 90.81px;
-  height: 130px;
-  left: 80%;
-  top: -220px;
-  background-image: url('img/illustration.png');
-  background-size: contain;
-  background-position: center center;
-  background-repeat: no-repeat;
-  z-index: 3;
-}
+  .container {
+    background-size: 90%;
+    height: 120vh;
+  }
 
-@media only screen and (min-width: 375px) {
-  body {
-    font-size: 16px;
+  .menu {
+    display: flex;
   }
-}
 
-@media only screen and (min-width: 768px) {
-  body {
-    font-size: 25px;
+  .menu li {
+    padding: 10px;
   }
 
-  #about-id {
-    position: relative;
-    overflow: hidden;
-    margin: 0;
-    padding: 0;
+  .head {
+    margin-top: 186px;
   }
 
-  #about-id::before {
+  .me-container {
+    flex-direction: row;
+    gap: 40px;
+    align-content: baseline;
+  }
+
+  .box {
     display: flex;
-    justify-content: flex-end;
-    align-items: flex-end;
-    align-content: flex-end;
-    content: "";
-    position: absolute;
-    top: 0;
-    margin-left: 90%;
-    width: 68px;
-    height: 128px;
-    background-image: url('img/Mask Group.png');
-    background-repeat: no-repeat;
-    background-size: cover;
-    z-index: 1;
+    width: 100%;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: center;
+    gap: 20px;
+  }
+
+  .background {
+    width: 30%;
+    background-color: #d0d9d4;
+    border: 2px solid #c1c7d0;
+    border-radius: 8px;
+    margin-top: 24px;
   }
 
-  #about-id::after {
+  .card h2 {
+    font-size: 1.9em;
+  }
+
+  .card li {
+    font-size: 1em;
+  }
+
+  .a-container {
+    background: no-repeat top right / 16% url("Images/ill-1_about-me.png"), no-repeat bottom left / 16% url("Images/ill-2_about-me.png");
+  }
+
+  .about {
+    margin-top: 35px;
+  }
+
+  .a-container .text {
+    width: 70%;
+  }
+
+  .form-container {
+    flex-direction: column;
+    flex-wrap: nowrap;
+    gap: 10px;
+    background: no-repeat 0% 75% / 12.5% url("/Images/contact-desk1.png"), no-repeat 20% 70% / 6% url("/Images/contact-desk2.png"), no-repeat 100% 60% / 15%  url("/Images/contact-desk3.png");
+  }
+
+  .form-container > .text {
+    display: none;
+  }
+
+  #comment {
+    background-size: 5%;
+  }
+
+  .foot {
     display: flex;
-    justify-content: flex-start;
-    align-items: flex-start;
-    align-content: flex-start;
-    content: "";
-    width: 128.02px;
-    height: 57.04px;
-    background-image: url('img/Frame.png');
-    background-repeat: no-repeat;
-    background-size: contain;
-    z-index: 2;
+    width: 100%;
+    border-top: 2px solid #dfe1e6;
+    justify-content: center;
+  }
+
+  .foot > .socials {
+    margin-top: 5px;
```

Comment:
Nice job so far👍. Let's try to make some improvements to the project so it matches the given design😉  don't forget it is a requirement for this milestone that our project matches the design even as we make it responsive😉.

desktop

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img width="400" alt="image" src="https://github.com/NgachaIan/Personal_Portfolio/assets/64643995/7714de77-32e9-4492-b91d-adabdddc6390">|<img width="400" alt="image" src="https://github.com/NgachaIan/Personal_Portfolio/assets/64643995/c2340426-bd1e-47bd-8a5b-c29b6ee002e2">|

mobile

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img width="239" alt="image" src="https://github.com/NgachaIan/Personal_Portfolio/assets/64643995/d5ca229e-fc89-4b0d-b40d-bf96e6b1652a">|<img width="386" alt="image" src="https://github.com/NgachaIan/Personal_Portfolio/assets/64643995/253aab22-b267-4b47-9ce1-ac188f87a2eb">|

Kindly take note of the positioning of each element and the spacings around them😉.  also, see how the texts are positioned within the lines of the background image and also see the "scroll down" button and its position😉. kindly try to implement your headline section to match the design👍.

desktop

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img width="400" alt="image" src="https://github.com/NgachaIan/Personal_Portfolio/assets/64643995/af82b2b2-5009-4a39-b3a9-2fd5b8ffd067">|<img width="400" alt="image" src="https://github.com/NgachaIan/Personal_Portfolio/assets/64643995/d8ba855a-28ff-4618-8054-3eb8fb04e80c">|


for your contact section, let's try to take note of how the elements are positioned, and the spacings (margin and padding) around them and make the necessary adjustments to them😉.